components/feature/prompts/src/test/java/mozilla/components/feature/prompts/PromptFeatureTest.kt

```diff
@@ -632,10 +636,9 @@ class PromptFeatureTest {
             "message",
             "input",
             false,
+            { _, _ -> onConfirmWasCalled = true },
             { onDismissWasCalled = true }
```

Comment:
This is more of a note, but this is a good example of why we might want to prefer adding named parameters when the list of parameters are long, not obvious and ordering can change in subsequent issues.